docs/datamodel/aliases.rst

```diff
@@ -33,6 +33,27 @@ of all User objects*. After declaring the alias below, you can use ``User`` and
 
   alias UserAlias := User;
 
+**Tuple alias**
+
+A tuple alias has features in common with both scalar and object type aliases.
+Like object type aliases, a tuple alias can hold multiple values that can be
+accessed using the ``.`` dot operator. But like scalar aliases, a tuple alias
+is a standalone expression that does not involve object types.
```

Comment:
I'm trying to understand whether it's accurate to say that a tuple alias "does not involve object types." My hypothesis was that a tuple could _contain_ an object type and as a result maybe we need to use different phrasing here.

In researching this, I uncovered some InternalServerErrors, so I'm not sure what the intended behavior actually is. 😅  See #5887.

Reply:
Okay, as discussed looks like this would serve best as a single example inside the existing section on expression aliases. This works just as well since the desired outcome is just that the reader come away from the page knowing that aliases can be more than simple scalar values on one end and expressions on schema-based types on the other.

Reply:
The example feels disconnected from the text that precedes it. What if we replaced the "heading" and the sentence with this?
> **Expression alias**
> Aliases can correspond to arbitrary EdgeQL expressions, for example tuples or entire queries.

Then, within the SDL block, we could add comments that would label each of those.

```
# Tuple alias
…

# Named tuple alias
…

# Query alias
…
```

Reply:
Yeah. Rereading it as it currently stands it starts out by introducing the section as "Query alias" but then says that these aliases "can correspond...to entire queries" which reads a bit oddly since it's already a given by the title. So that looks good I think.

Except I think I'll change the "Tuple alias" and such comments to "Expression producing a tuple alias" which is longer but no risk of giving the impression that we're introducing a strict set of possible aliases and that others aren't possible. Want to keep the sky's the limit feel.

Reply:
Although...reading this again the whole page starts by describing aliases as expressions. So maybe instead of Expression alias the title should be a bit different. Like "Other arbitrary alias examples".